libc-test/build.rs

```diff
@@ -281,6 +281,9 @@ fn main() {
         match ty {
             "sockaddr_nl" => musl,
 
+            // user-defined struct to patch struct utmpx on Linux
+            "__timeval" if linux => true,
```

Comment:
Could you elaborate a bit on this exception? How come we can't match what libc does?